clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+## Configure SSL trust store
+
+Depending on your operating system or Linux distro you may need to take extra steps to set up the SSL CA root certificates.
+
+### CentOS
+
+```bash
+$ sudo yum reinstall ca-certificates
+```
+
+Add the following property to the config dict objects in `producer.py` and `consumer.py`:
```

Comment:
@edenhill do other librdkafka clients have this issue (and thus the other examples in this repo have the same problem)?

Reply:
Yes, the .NET client might.
But not librdkafka and Go since they're typically built using the distros openssl packages which also installs the certs.